### PR TITLE
new fee window unit tests. added and modified mocks, removed unneeded…

### PR DESCRIPTION
**Pull request by bthaile**
removed a few unneeded contract references, added fee window unit tests, modified/added mocks.

### CODE REVIEW

File: source/contracts/reporting/FeeWindow.sol [hunk 1]
```diff
@@ -80,13 +79,11 @@ contract FeeWindow is DelegationTarget, VariableSupplyToken, Extractable, Initia
     }
 
     function redeemForReportingParticipant() public returns (bool) {
-        redeemInternal(msg.sender, true);
-        return true;
+        return redeemInternal(msg.sender, true);
```
Comment (nuevoalex) [inline source/contracts/reporting/FeeWindow.sol]: I'd prefer these stay as is actually. Making it this way implies there's some information being passed.

File: tests/solidity_test_helpers/MockReputationToken.sol [hunk 2]
```diff
@@ -9,7 +9,7 @@ import 'reporting/IUniverse.sol';
 import 'reporting/IMarket.sol';
 import 'reporting/Reporting.sol';
 import 'libraries/math/SafeMathUint256.sol';
-import 'TEST/MockVariableSupplyToken.sol';
+import 'TEST/MockVariableSupplyToken.sol';import 'TEST/MockVariableSupplyToken.sol';
```
Comment (nuevoalex) [inline tests/solidity_test_helpers/MockReputationToken.sol]: typo I think

File: source/contracts/reporting/FeeWindow.sol [hunk 3]
```diff
@@ -108,7 +105,7 @@ contract FeeWindow is DelegationTarget, VariableSupplyToken, Extractable, Initia
         }
 
         if (_totalFeeStake == 0) {
-            return;
+            return true;
```
Comment (MicahZoltu) [inline source/contracts/reporting/FeeWindow.sol]: 😡 It makes me so angry that Solidity compiler didn't error on this previously... good catch.

File: tests/unit/test_fee_window.py [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+def mockFeeTokenFactory(localFixture):
+    return localFixture.contracts['MockFeeTokenFactory']
+
+@fixture
+def mockInitialReporter(localFixture):
+    return localFixture.contracts['MockInitialReporter']
+
+@fixture
+def Time(localFixture):
+    return localFixture.contracts["Time"]
```
Comment (nuevoalex) [inline tests/unit/test_fee_window.py]: newline at end of file please. You can setup your editor to do this automatically most likely.

Comment (bthaile) [reply tests/unit/test_fee_window.py]: Thanks, installed 'final-newline' extension into vscode.

**Pull request merged**